ci/daily-snapshot.yml

```diff
@@ -3,13 +3,13 @@
 
 pr: none
 trigger: none
-schedules:
-- cron: "0 1 * * *"
-  displayName: Daily split snapshot
-  branches:
-    include:
-    - main
-    - main-2.x
+schedules: none
+#- cron: "0 1 * * *"
+#  displayName: Daily split snapshot
+#  branches:
+#    include:
+#    - main
+#    - main-2.x
```

Comment:
I think it will be less disruptive to not have a new snapshot for a couple days than to have one that doesn't have macOS artifacts.